docs/configuration.md

```diff
@@ -125,14 +125,24 @@ Check out the [babel-core documentation](core.md) to see more configuration opti
 
 You can tell Babel to print effective configs on a given input path
 
-```sh title="Shell"
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="shell" label="Shell" default>
+<CodeBlock language="bash">{`
 # *nix or WSL
 BABEL_SHOW_CONFIG_FOR=./src/myComponent.jsx npm start
-```
-
-```powershell
+`}
+</CodeBlock>
+</TabItem>
+<TabItem value="powershell" label="powershell">
+<CodeBlock language="powershell">{`
 $env:BABEL_SHOW_CONFIG_FOR = ".\src\myComponent.jsx"; npm start
-```
+`}</CodeBlock>
+</TabItem>
+</Tabs>
```

Comment:
See https://deploy-preview-2726--babel.netlify.app/docs/configuration#print-effective-configs for preview.